src/locale/en-US.json

```diff
@@ -125,7 +128,7 @@
   "Quit Symphony": "Quit Symphony",
   "Redo": "Redo",
   "Refresh app when idle": "Refresh app when idle",
-  "Relaunch": "Relaunch",
```

Comment:
@VishwasShashidhar we need this for https://github.com/symphonyoss/SymphonyElectron/pull/884/files#diff-aed34bfdf22162ca801ca8365863501aR159

Reply:
Done @KiranNiranjan, fixed.